webrender/src/platform/macos/font.rs

```diff
@@ -358,7 +360,31 @@ impl FontContext {
                 let glyph = key.index as CGGlyph;
                 let bitmap = is_bitmap_font(ct_font);
                 let (x_offset, y_offset) = if bitmap { (0.0, 0.0) } else { font.get_subpx_offset(key) };
-                let metrics = get_glyph_metrics(ct_font, None, glyph, x_offset, y_offset, 0.0);
+                let mut shape = FontTransform::identity();
```

Comment:
I don't think we need to create ```shape``` here unconditionally. It's probably tidier to make the ```let transform``` line below be based on the ```flags``` rather than checking the shape matrix is identity.